pkg/api/v1/defaults.go

```diff
@@ -372,3 +372,17 @@ func SetDefaults_ScaleIOVolumeSource(obj *v1.ScaleIOVolumeSource) {
 		obj.FSType = "xfs"
 	}
 }
+
+func SetDefaults_QuobyteVolumeSource(obj *v1.QuobyteVolumeSource) {
+	if obj.User == "" {
+		obj.User = "root"
```

Comment:
Exactly the same behavior we just handled the defaults inside the volume package: https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/quobyte/quobyte.go#L272-L280